content/docs/getting-started/installation.md

```diff
@@ -111,14 +111,6 @@ choco install ocm-cli
 
 see: [chocolatey community package: ocm-cli](https://community.chocolatey.org/packages/ocm-cli)
 
-### using winget
```

Comment:
please replace this with a section that mentions that we used to publish to winget but no longer do so